components/dashboard/src/App.tsx

```diff
@@ -372,6 +373,7 @@ function App() {
                     <AdminRoute path="/admin/license" component={License} />
                     <AdminRoute path="/admin/settings" component={AdminSettings} />
 
+                    <Route path="/feedback" exact component={FeedbackFormModal} />
```

Comment:
**nitpick**: Friendly reminder to remove this route as we no longer need it, right?

Reply:
Ohh thank you for this!!

Reply:
Done